_shows/16_17/wrecked_edin.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Wrecked
+devised: true
+period: Edinburgh
+season: Edinburgh
+season_sort: 580
+date_start: 2017-08-04
+date_end: 2017-08-26
+venue: Royal Terrace, Greenside
+venue_sort: Greenside
+
+assets:
+  - type: poster
+    image: t6mfPwb
+
+links:
+  - type: Review
+    href: http://edfringereview.com/r/WYh06bDwTo6x5kwRXDx6yg?
+    snapshot: fKFbP
+    publisher: edfringereview
```

Comment:
Where should spacing be? The website header has it as 'edfringe review' while the about page has 'Ed Fringe Review'

Reply:
Also take that `?` out